### PR TITLE
feat: set external entities as not draggable (Issue #584)

### PR DESCRIPTION
**Pull request by IlyaBondar**
- Issue #584

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

File: src/utils/app/share.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const getUnpublishActionByType = (type: SharingType) => {
   }
 };
 
-export const isEntityExternal = (entity: ShareEntity) =>
+export const isThisEntityExternal = (entity: ShareEntity) =>
```
Comment (mikitabut) [inline src/utils/app/share.ts]: What is the difference between isThisEntityExternal and isExternalEntity? 
It's not clear when to use them
Is it possible to rename somehow?

Comment (mikitabut) [reply src/utils/app/share.ts]: Maybe it's possible to use full check for external everywhere?

Comment (IlyaBondar) [reply src/utils/app/share.ts]: @mikitabut , I specially added `This` word to have difference when we check entity itself only (and when we check entity and parents)

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Review by mikitabut — APPROVED** [review]

**Pull request merged**